com.unity.netcode.gameobjects/Runtime/NetworkVariable/NetworkVariableBase.cs

```diff
@@ -94,6 +94,14 @@ public bool CanClientWrite(ulong clientId)
             }
         }
 
+        /// <summary>
+        /// Returns the ClientId of the owning client
+        /// </summary>
+        internal ulong OwnerClientId()
+        {
+            return m_NetworkBehaviour.NetworkObject.OwnerClientId;
+        }
```

Comment:
not needed?

Reply:
if anything, aggressiveinline attr would also be useful too if we'd like to keep this around.

Reply:
Actually, this is needed as `m_NetworkBehaviour` is `private` to the `networkVariable`. 

You know I come from C++ and don't know too much about C#. But... seriously? This doesn't require serious inlining. It's rather basic compiler 101. If this doesn't get optimized out... 🤯 . And it feels like we are much more at the iron-out-the-bugs phase, than optimize-the-crap-out.

I'm ok with inlining it in and removing the function call if you make `m_NetworkBehaviour` `internal` instead of `private`. But it feels wrong.

Reply:
aggressive inline attribute is just a hint for the compiler, not like "absolutely inline this" but also at the same time, without the attribute, compiler _might_ inline it on its own too — we're just trying to help compiler here :)